engine/src/main/java/org/pentaho/di/kitchen/KitchenCommandExecutor.java

```diff
@@ -26,6 +26,7 @@
 import org.pentaho.di.base.AbstractBaseCommandExecutor;
```

Comment:
Update the copyright date on this and other files

Reply:
updated